internal/broker/broker.go

```diff
@@ -634,6 +634,7 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, session *session, au
 	}
 
 	if !b.userNameIsAllowed(authInfo.UserInfo.Name) {
+		log.Errorf(context.Background(), "User %q is not in the list of allowed users", authInfo.UserInfo.Name)
```

Comment:
This is the only one that I'm unsure about... As IIRC we didn't want to give much information (neither in log form) whether someone was or not managed by the broker.

But I feel that's not a too big issue, given that normally users don't have access to the journal... That said, I'd leave the decision to @didrocks.

Reply:
As I mentioned in the commit message, it's considered a security risk to *not* log failed login attempts:

> To 1. allow the user/admin to figure out why authentication is denied,
and 2. log failed login attempts, which is security critical (security
logging failure is in the OWASP top 10, see
https://owasp.org/Top10/A09_2021-Security_Logging_and_Monitoring_Failures).

Reply:
@didrocks @3v1n0: I would like to include that in the upcoming release, so let's please try to move this forward

Reply:
yeah, I think this is not a big issue by itself. I would though downgrade the log to a warning: this is not an error of the service itself (something malfunctionning), but something to alert the sysadmin on if looking at reasons why something didn’t go to completion.

Reply:
Ah, good point! I see that `pam_unix` prints it's authentication failure messages with priority `notice`, which is between `info` and `warning`:

```
sudo[33720]: pam_unix(sudo:auth): authentication failure; logname=user uid=1000 euid=0 tty=/dev/pts/8 ruser=user rhost=  user=user
sshd[33337]: pam_unix(sshd:auth): authentication failure; logname= uid=0 euid=0 tty=ssh ruser= rhost=127.0.0.1  user=user
```

I'll do the same if you agree.

Reply:
It requires supporting the notice level in the `log` package in authd. I'll do that in a follow-up and use log level `warning` for now.

Reply:
done

Reply:
> I'll do that in a follow-up and use log level warning for now.

tracked in https://warthogs.atlassian.net/browse/UDENG-5879